lib/maths/CSeasonalTime.cc

```diff
@@ -15,10 +16,9 @@
 #include <maths/CChecksum.h>
 #include <maths/CIntegerTools.h>
 
-#include <boost/array.hpp>
```

Comment:
It looks like this file had already been changed to `std::array` on master without changing the header, so this is fine on master.  But note that on 6.8 the code is still using `boost::array`, so the header change shouldn't be included in the 6.8 backport.

Reply:
Yes, I need to rewrite the tests as well on 6.8 so the back port is non-trivial.